lib/modules/newCommentCount.js

```diff
@@ -385,8 +385,13 @@ function updateCommentCount(mycomment) {
 async function cleanCache() {
 	await getLatestCommentCounts();
 	const now = Date.now();
+	const skip = module.options.cleanComments.value < module.options.subscriptionLength.value;
```

Comment:
This is correct, but I think we should leave this out and ignore all comment counts where `subscriptionDate` is defined, regardless of `subscriptionLength`. That way the logic for removing thread subscriptions and removing comment counts are kept separate.